src/page-form/components/TextInput.tsx

```diff
@@ -0,0 +1,70 @@
+import PropTypes from "prop-types";
+import React from "react";
+import styled from "styled-components";
+
+const Label = styled.span`
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 100;
+  font-family: "Poppins", sans-serif;
+  letter-spacing: 2px;
+  color: #00413e;
+`;
+
+const Input = styled.input`
+  padding: 16px;
+  background: #e0e4e4;
+  outline: 0 solid transparent;
+  box-shadow: none;
+  border: none;
+  font-size: 16px;
+  border-radius: 2px;
+  font-family: "Rubik", sans-serif;
+  color: #00413e;
+  margin-top: 8px;
+`;
+
+const TextInputContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 24px;
+`;
+
+interface Props {
+  label: string;
+  placeholder?: string;
+  value?: string;
+  onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  defaultValue?: string;
+  type?: string;
+}
+
+const TextInput: React.FC<Props> = (props) => {
+  return (
+    <TextInputContainer>
+      <Label>{props.label}</Label>
+      <Input
+        type={props.type}
+        defaultValue={props.defaultValue}
+        value={props.value}
+        placeholder={props.placeholder}
+        onChange={props.onChange}
+      />
+    </TextInputContainer>
+  );
+};
+
+TextInput.propTypes = {
+  label: PropTypes.string.isRequired,
+  placeholder: PropTypes.string,
+  value: PropTypes.string,
+  onChange: PropTypes.func,
+  defaultValue: PropTypes.string,
+  type: PropTypes.string,
+};
+
+TextInput.defaultProps = {
+  type: "text",
+};
```

Comment:
I think PropTypes is largely redundant with TypeScript props interfaces -- we can probably just not use PropTypes?

As for defaultProps, we can do `type={props.type || "text"}` above. Or leave it, since I think having defaults can sometimes hide complexity in subtle ways.

Reply:
Oh, I see that the current eslint config enforces it.

We can remove that rule from eslint! Doing so in https://github.com/Recidiviz/covid19-dashboard/pull/5#discussion_r399651406

Let me know if you actually wanted to continue using prop types.

Reply:
Feel free to just 

```
git fetch && git checkout origin/szhu/news-feed .eslintrc.json
```

if you want it disabled before #5 is merged.